shared/src/types.ts

```diff
@@ -2,3 +2,8 @@ export enum TreeMode {
   forEachChild,
   getChildren,
 }
+
+export enum Theme {
+  Dark = "dark",
+  Light = "light",
```

Comment:
Would be neat to add a "system" setting in the future maybe.

Reply:
I'm up for doing that, will give it a shot

Reply:
Done! First time I've messed with `useReducer`, but it seemed like the right tool for the job in this case...

Reply:
I wrote this code originally when redux was a thing in order to try it out then switched it to useReducer when upgrading to hooks. I wouldn't use it again tbh, but I'm not really a web developer.